x-pack/plugins/security_solution/public/management/components/artifact_list_page/components/artifact_flyout.tsx

```diff
@@ -315,6 +347,7 @@ export const MaybeArtifactFlyout = memo<ArtifactFlyoutProps>(
               item={formState.item}
               error={submitError ?? undefined}
               mode={formMode}
+              policiesRequest={policiesRequest}
```

Comment:
I would suggest not passing along the entire request. only the array of policies